microsoft-graph/release_notes.md

```diff
@@ -1,10 +1,10 @@
 #### What's Improved
-- Security alert-related actions now support API versions V1 and V2. 
-  - Added a new parameter `API Version` with support options V1 and V2 to the following actions
+- Security alert-related actions now support API versions V1 and V2.
+  - Added a new `API Version` parameter with support for options V1 and V2, and added conditional output schema support based on this parameter to the following actions
```

Comment:
Added a new `API Version` parameter with support for options V1 and V2, and added conditional output schema support based on this parameter to the following actions ---> Added a new `API Version` parameter to support  API versions v1 and v2. Also added conditional output schema support based on this parameter to the following actions